R/read_clip_tbl.R

```diff
@@ -0,0 +1,30 @@
+#' Transforms Output of Read_clip into Data.Frame
+#'
+#' Transforms Output of Read_clip into Data.Frame
```

Comment:
This line should not repeat, but instead contain the more detailed description that you've currently put in README.md.